smarts/core/lidar.py

```diff
@@ -90,8 +89,44 @@ def _compute_rays(self):
             for yaw, roll in itertools.product(yaws, rolls):
                 rot = pybullet.getQuaternionFromEuler((roll, 0, yaw))
                 origin = np.array([0, 0, 0])
-                direction = np.array(
-                    Quat(rot).xform((0, self._sensor_params.max_distance, 0))
+
+                def mult_quat(q1, q2):
```

Comment:
I could suggest moving this and the quaternion rotation method to `utils/math`.

Reply:
oh ok, will do.